dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ExpressionDimensionItemController.java

```diff
@@ -42,6 +50,33 @@
 @Controller
 @RequestMapping( value = ExpressionDimensionItemSchemaDescriptor.API_ENDPOINT )
 @ApiVersion( { DhisApiVersion.DEFAULT, DhisApiVersion.ALL } )
+@AllArgsConstructor
 public class ExpressionDimensionItemController extends AbstractCrudController<ExpressionDimensionItem>
 {
+    private final AnalyticsCache analyticsCache;
+
+    private final ExpressionDimensionItemAnalyticsCache expressionDimensionItemAnalyticsCache;
+
+    /**
+     * Extension of basic method is handling of update event for expression
+     * dimension item
+     *
+     * @param patch
+     * @param entityAfter
+     */
+    @Override
+    protected void postPatchEntity( JsonPatch patch, ExpressionDimensionItem entityAfter )
```

Comment:
I believe you could call this argument `expressionDimensionItem`, `entityAfter` sounds strange. We already know this is a post-action, based on the method `postPatchEntity`